tools/devtool

```diff
@@ -73,7 +73,7 @@
 # Development container image (name:tag)
 # This should be updated whenever we upgrade the development container.
 # (Yet another step on our way to reproducible builds.)
-DEVCTR_IMAGE="fcuvm/dev:v10"
+DEVCTR_IMAGE="fcuvm/dev:v13"
```

Comment:
What happened to v11 and v12?

Reply:
I botched them as part of the learning process of publishing a new container version